src/Core/Components/Camera/CameraLensFlare.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "Utilities/ECS/Component.h"
+
+
+namespace MxEngine
+{
+	class CameraLensFlare 
+	{
+        MAKE_COMPONENT(CameraLensFlare);
+    private:
+        float lensFlareScale = 1.f;
+        float lensFlareBias = -42;
+        int lensFlareNumOfGhosts = 10;
+        float lensFlareGhostDispersal = 0.13f;
+        float lensFalreHaloWidth = 0.5f;
+        bool lensFlareEnable = true;
```

Comment:
I guess it is redundant. Existance of component ib object should already mean that effect is enabled